packages/plugin-assets-retry/src/AssetsRetryPlugin.ts

```diff
@@ -33,6 +35,7 @@ export class AssetsRetryPlugin implements Rspack.RspackPluginInstance {
       distDir,
       HtmlPlugin,
       inlineScript = true,
+      minify = true,
```

Comment:
How about only enable minify in production:

```suggestion
      minify = process.env.NODE_ENV === 'production',
```